src/liapunovspectrum.jl

```diff
@@ -169,9 +173,16 @@ function liap_taylorinteg(f, q0::Array{U,1}, t0::T, tmax::T,
     const λ = similar(xv)
     const λtsum = similar(q0)
     const jt = eye(U, dof)
+    const _δv = Array{TaylorN{Taylor1{U}}}(dof)
 
-    # NOTE: This changes GLOBALLY internal parameters of TaylorN
-    global _δv = set_variables("δ", order=1, numvars=dof)
+    if get_numvars() == dof
+        for ind in eachindex(q0)
+            _δv[ind] = TaylorN(Taylor1{U},ind,order=1)
+        end
+    else
```

Comment:
A question here: if the `else` applies, should we throw an error (DomainError, or something like that)? My point
is that the user should be responsible of this subtlety...

Same comment applies below.